trulens_eval/trulens_eval/feedback/groundtruth.py

```diff
@@ -198,8 +198,8 @@ def mae(self, prompt: str, response: str, score: float) -> float:
         """
 
         expected_score = self._find_score(prompt, response)
-        if expected_score:
-            ret = abs(float(score) - expected_score)
+        if expected_score is not None:
```

Comment:
good catch!

Reply:
Since you are touching up / fixing this method, do you mind renaming it to something more accurate like `mae_term`? 

I noticed the method itself is actually calculating the absolute difference b/w ground truth and generated score and it's a misnomer / not MAE without the `aggregate(np.mean)` part.

Reply:
Renamed to `absolute_error` and updated the name in notebooks as well